scripts/node.sh

```diff
@@ -101,17 +108,25 @@ fi
 # find my public ip address
 myip
 
-# public beacon node multiaddress
-BC_MA=/ip4/54.183.5.66/tcp/9999/ipfs/QmdQVypu6NSm7m8bNZj5EJCnjPhXR8QyRmDnDBidxGaHWi
+# public boot node multiaddress
+BN_MA=/ip4/100.26.90.187/tcp/9876/p2p/QmZJJx6AdaoEkGLrYG4JeLCKeCKDjnFz2wfHNHxAqFSGA9,/ip4/54.213.43.194/tcp/9876/p2p/QmQayinFSgMMw5cSpDUiD9pQ2WeP6WNmGxpZ6ou3mdVFJX
 
 if [ "$OS" == "Linux" ]; then
 # Run Harmony Node
-   nohup ./harmony -bc_addr $BC_MA -ip $PUB_IP -port $NODE_PORT > harmony-${PUB_IP}.log 2>&1 &
+   LD_LIBRARY_PATH=$(pwd) nohup ./harmony -bootnodes $BN_MA -ip $PUB_IP -port $NODE_PORT -is_beacon > harmony-${PUB_IP}.log 2>&1 &
```

Comment:
A [DRY](https://en.wikipedia.org/wiki/Don%27t_repeat_yourself)-refactor is due here.  How about adding a reusable script `run_with_libpath.sh`:

```
#!/bin/sh
set -eu
unset -v libpath_var libpath
libpath_var=LD_LIBRARY_PATH  # most platforms use this
case "$(uname -s)" in
Darwin) libpath_var=DYLD_FALLBACK_LIBRARY_PATH;;
esac
libpath="${1-}"
shift 1 || { echo "run_with_libpath: libpath not given" >&2; exit 64; }
exec env "${libpath_var}=${libpath}" "$@"
```

and then using it from `node.sh`:

```
# Add this around the beginning of the script
unset -v progname progdir
progname="${0##*/}"
case "${0}" in
*/*) progdir="${0%/*}";;
*) progdir=.;;
esac
```

```
# Substitute this for the if-then-else-fi
"${progdir}/run_with_libpath.sh" "$(pwd)" nohup ./harmony -bootnodes $BN_MA -ip $PUB_IP -port $NODE_PORT -is_beacon > harmony-${PUB_IP}.log 2>&1 &
```